readme.md

```diff
@@ -35,35 +40,221 @@ npm install nano-spawn
 
 ## Usage
 
-```js
-import $ from 'nano-spawn';
+### Run commands
 
-const result = await $('echo', ['🦄']);
+```js
+import spawn from 'nano-spawn';
```

Comment:
Calling it `spawn` instead of `$` because `$` might be a better fit for scripts, while this package is meant for programmatic users. Namely, in a programmatic usage, there is not really a need for short variable names.

Also, the API is very similar to `node:child_process` `spawn()`, so using the same function name might make sense.